### ISSUE TITLE
User cannot run code in "Run analysis" vignette

### ISSUE THREAD

**Issue by ellisifnygaard** [jromanowska/HaplinMethyl#19]
### TL;DR
Not sure if this qualifies as a bug, so just remove the tag if you disagree 🐞 

There are a few things that I noticed that might be worth addressing in order to ensure that the vignette is a useful resource for users trying to get familiar with the package:

1. The vignette uses simulated genotype data that is not available to users via the package.
2. When using the exact same code as in [make_exemplary_data.R](https://github.com/jromanowska/HaplinMethyl/blob/7fc4afdc0c45f38321896fb55323bf1d54dc9372/inst/make_examplary_data.R#L48-L69) to create example genotype data, the data is not identical to the data shown in the vignette.
3. When running the analysis from the vignette using the example genotype data descrided above, the resulting haptable is different from the one in the vignette.

### More detailed description of issue
I just tried running the code in the [Run analysis vignette](https://jromanowska.github.io/HaplinMethyl/articles/Run_analysis.html) while while testing out some stuff for the pipeline.

I manage to load this data just fine:
```
ex_path <- system.file("extdata", package = "HaplinMethyl")
ex_file <- "env_data_test.dat"
ex_out_file <- "dnam_ex"

dnam_ex <- envDataRead(
  file.in = ex_file,
  dir.in = ex_path,
  file.out = ex_out_file,
  sep = " ",
  overwrite = TRUE
)
```
But this data file does not appear to exist anywhere in the package directory:

```
ex_gen_file <- "sim1"
gen_ex <- genDataLoad(
  filename = ex_gen_file,
  dir.in = ex_path
)
```
I tried running ` system.file("sim1", package = "HaplinMethyl")` and searching the directory on GitHub for files names "sim1", and it does not appear to be available.

I ended up recreating the data using the code from [make_exemplary_data.R](https://github.com/jromanowska/HaplinMethyl/blob/7fc4afdc0c45f38321896fb55323bf1d54dc9372/inst/make_examplary_data.R#L48-L69), but the genotype data was not identical to the printed output in the vignette. I was able to successfully run the example from the vignette though, just with different results. 

Also, my haptable had fewer column, and some RR columns with different names , compared to the printout in the vignette (se screenshot of the haptable below). The haptable I got did not have column names containing `cm` and `cf` like in `RRcm.est.` etc. 
Is this due to changes made to `haptable()` after this vignette was made, the fact that my genotype data was different, or some other reason?

![image](https://github.com/jromanowska/HaplinMethyl/assets/63318842/3db1ee21-dddc-41d3-bba7-3d0e5e84e1ac)

#### R session info
(This was done on SAFE, so I'll just post the versions of relevant packages)
`HaplinMethyl_0.2`
`Haplin_7.3.0`
`ff_4.0.9`

**Comment by jromanowska** [comment]
This was fixed in [f1c6cdf3ebd3f1007c356ee1effc1e816fe025b4](https://github.com/jromanowska/HaplinMethyl/commit/f1c6cdf3ebd3f1007c356ee1effc1e816fe025b4)
